src/app/components/url-box/url-box.component.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, EventEmitter, Input, Output, ViewChild} from '@angular/core';
+
+@Component({
+  selector: 'app-url-box',
+  templateUrl: './url-box.component.html'
+})
+export class UrlBoxComponent {
+  @Input() apiUrl: string;
+  @Input() verb: string;
+
+  @Output() updatedUrl = new EventEmitter();
```

Comment:
The naming convention for output variables should have the verb (usually `change`) behind. In this case, you would have `urlChange` and `httpVerbChange` as the event emitters.

Reply:
That way it's easy to identify and group them.